infrastructure/terraform/components/iam/module_amplify_branch.tf

```
@@ -1,21 +1,21 @@
 module "amplify_branch" {
-  source = "../../modules/amp_branch"
-
-  name           = "main"
-  aws_account_id = var.aws_account_id
-  component      = var.component
-  environment    = var.environment
-  project        = var.project
-  region         = var.region
-  group          = var.group
-
-  cognito_user_pool_client_id               = aws_cognito_user_pool_client.main.id
-  cognito_user_pool_identity_provider_names = aws_cognito_user_pool_client.main.supported_identity_providers
-  amplify_app_id                            = aws_amplify_app.main.id
-  branch                                    = "main"
-  domain_name                               = local.root_domain_name
-  subdomain                                 = var.environment
-  base_path                                 = "/auth"
-  enable_auto_deploy                        = true
-  stage                                     = "PRODUCTION"
+  source            = "git::https://github.com/NHSDigital/nhs-notify-shared-modules.git//infrastructure/modules/amp_branch?ref=v1.0.0"
```

put an empty line after the meta attributes like source,
```suggestion
source = "git::https://github.com/NHSDigital/nhs-notify-shared-modules.git//infrastructure/modules/amp_branch?ref=v1.0.0"

```